app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughTopicListFragmentTest.kt

```diff
@@ -106,7 +106,7 @@ class WalkthroughTopicListFragmentTest {
   }
 
   @Test
-  fun testWalkthroughWelcomeFragment_recyclerViewIndex0_topicHeader_topicHeaderIsCorrect() {
+  fun testWalkthroughTopicListFragment_topicHeader_whatDoYouWantToLearnIsDisplayed() {
```

Comment:
will changing the name to `testWalkthroughTopicListFragment_topicHeaderSelected_whatDoYouWantToLearnIsDisplayed()` be meaningful?

Reply:
keeping `testWalkthroughTopicListFragment_topicHeader_whatDoYouWantToLearnIsDisplayed` is also sounds good.
Is there anything blocking in this?

Reply:
no there is nothing blocking this but since we are checking the topic header on recycler view index 0 I thought it would be better to use topicHeaderSelected